src/config/dynamic-theme-fixes.config

```diff
@@ -8267,6 +8236,46 @@ INVERT
 
 ================================
 
+whatsapp.com
+
+INVERT
+.selectable-text > span._3PxOr
```

Comment:
This can be removed as it's been taken over `.landing-main .invisible-space > span`